esss_fix_format/cli.py

```diff
@@ -165,7 +165,19 @@ def _process_file(filename, check, format_code):
             changed = b'<replacement ' in output
         else:
             mtime = os.path.getmtime(filename)
-            subprocess.check_output('clang-format -i "%s"' % filename, shell=True)
+            
+            # checking if the terminal command is valid
+            # might throw an exception if clang-format is not recognized
+            try:
+                subprocess.check_output('clang-format -i "%s"' % filename, shell=True)
+            except subprocess.CalledProcessError as e:
+                errors.append(": ERROR")
```

Comment:
This was a left over? A better error message is added to this list in `errors.append(error_msg)`.

Reply:
Yes, I'll remove it :)